### DISCUSSION TITLE
Is there a way of generating a number of images from a set of random prompt?

### DISCUSSION THREAD

**Discussion by msqiqi10** [adieyal/sd-dynamic-prompts#510] [Q&A]
Say I have __Extremely beautiful clothes__, and if I want to generate 10 images instead of 1 image from each prompt randomly chosen from the wildcard?
Is there a way of doing that?

**Comment by akx** [comment]
Set the Batch count to 10 and ensure you _don't_ have "Combinatorial generation" on (and haven't set a specific seed).

**Reply by msqiqi10** [reply]
I'm sorry but did you mean the "batch count" in t2i settings instead of Combinatorial batches right? That would result in a OOM in my device. Is there another way of doing it, like using loops? 
Thanks for the help!
